app/views/council_memberships/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>Add member to </h1>
+<%= form_for @council_membership, url: {action: 'create'} do |f| %>
+      <select name="council_membership[council_id]">
```

Comment:
You shouldn't need the URL parameter here. As long as @council_membership is new, rails knows to post against the create action.